barman/retention_policies.py

```diff
@@ -455,8 +455,10 @@ def retention_policy(self):
         return self.server.config.retention_policy
 
     @property
-    def available_backups(self):
-        return self.server.get_available_backups(BackupInfo.STATUS_NOT_EMPTY)
+    def available_backups(self, ignore_incrementals):
```

Comment:
i guess properties cannot have parameters directly, right? So should i create another property available_full_backups? Or any ideas here?

Reply:
Yeah, we cannot have parameters in properties :s